Sources/DDGSync/internal/AccountManager.swift

```diff
@@ -137,6 +137,25 @@ struct AccountManager: AccountManaging {
                                deviceType: account.deviceType)
     }
 
+    func deleteAccount(_ account: SyncAccount) async throws {
+        guard let token = account.token else {
+            throw SyncError.noToken
+        }
+
+        let devices = try await fetchDevicesForAccount(account)
+
+        // Logout other devices first or the call will fail
+        for device in devices.filter({ $0.id != account.deviceId }) {
+            try await logout(deviceId: device.id, token: token)
```

Comment:
Theoretically for multiple devices we could call `logout` in parallel (via task group). May be worth considering, but ultimately this isn’t going to be used frequently so I’ll leave it up to you whether to change it.

Reply:
Can optimise that later :)